src/features/generic/time-utils.ts

```diff
@@ -63,9 +63,12 @@ namespace TimeUtilities {
    * @returns a message based on the worked time and the percentage of billable hours
    */
   export const calculateWorkedTimeAndBillableHours = (user: TimeEntryTotalDto | DailyCombinedData) => {
-    const { total, projectTime, expected } = user;
+    const { total, projectTime, logged } = user;
 
-    const billableHoursPercentage = (projectTime/expected * 100).toFixed(1);
+    let billableHoursPercentage = (projectTime/logged * 100).toFixed(0);
+    if(logged === 0){
+      billableHoursPercentage = "0";
+    }
```

Comment:
You can use ternary operators (? and :) to simplify if else checks. Basic idea is that you check for something, if the check is true then the first block is assigned to variable otherwise the second block is assigned to the variable.
```suggestion
    const billableHoursPercentage = logged === 0 ? "0" : (projectTime/logged * 100).toFixed(0);
```

Reply:
Done